src/main/java/org/jenkinsci/plugins/saml/SamlEncryptionData.java

```diff
@@ -107,6 +108,14 @@ public boolean isForceSignRedirectBindingAuthnRequest() {
         return forceSignRedirectBindingAuthnRequest;
     }
 
+    public boolean isWantsAssertionsSigned() {
+        return wantsAssertionsSigned;
+    }
+
+    public void setWantsAssertionsSigned(boolean wantsAssertionsSigned) {
+        this.wantsAssertionsSigned = wantsAssertionsSigned;
+    }
+
```

Comment:
```suggestion
```

Unused, unless you actually are going to switch to `@DataBoundSetter` (meaning, revert change to `@DataBoundConstructor` and tests).

Reply:
I know, all setters can be removed